source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc

```diff
@@ -37,20 +37,17 @@ All hosts in the cluster must use IPv4 or IPv6 for RHV networks, not simultaneou
 
 For details about IPv6 support, see link:{URL_virt_product_docs}{URL_format}administration_guide/index#IPv6-networking-support-labels[IPv6 Networking Support] in the Administration Guide.
 |NoVNC console option |Option for opening a virtual machine console in the browser using HTML5.
-|Websocket proxy |Allows users to connect to virtual machines through a noVNC console.
-|VDSM hook for nested virtualization |Allows a virtual machine to serve as a host. For details, see link:{URL_virt_product_docs}{URL_format}administration_guide/index#proc-enabling-nested-virtualization-for-all-virtual-machines[Enabling nested virtualization for all virtual machines] in the Administration Guide.
 |Import Debian and Ubuntu virtual machines from VMware and RHEL 5 Xen a|Allows `virt-v2v` to convert Debian and Ubuntu virtual machines from VMware or RHEL 5 Xen to KVM.
 
 *Known Issues:*
 
 * `virt-v2v` cannot change the default kernel in the GRUB2 configuration. The kernel configured on the guest operating system is not changed during the conversion, even if a more optimal version is available.
 
 * After converting a Debian or Ubuntu virtual machine from VMware to KVM, the name of the virtual machine's network interface may change, and will need to be configured manually
-|NVDIMM host devices |Support for attaching an emulated NVDIMM to virtual machines that are backed by NVDIMM on the host machine. For details, see link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#conc-nvdimm-host-devices_vm_guide_administrative_tasks[NVDIMM Host Devices].
-|Open vSwitch (OVS) cluster type support |Adds Open vSwitch networking capabilities.
 |Shared and local storage in the same data center |Allows the creation of single-brick Gluster volumes to enable local storage to be used as a storage domain in shared data centers.
-|Cinderlib Integration |Leverage CinderLib library to use Cinder-supported storage drivers in {virt-product-fullname} without a Full Cinder-OpenStack deployment. Adds support for Ceph storage along with Fibre Channel and iSCSI storage. The Cinder volume has multipath support on the {hypervisor-fullname}.
 |SSO with OpenID Connect |Adds support for external OpenID Connect authentication using Keycloak in both the user interface and with the REST API.
 |oVirt Engine Backup |Adds support to back up and restore {virt-product-fullname} {engine-name} with the Ansible `ovirt-engine-backup` role.
-|Failover vNIC profile | This feature allows users to migrate a virtual machine connected via SR-IOV with minimal downtime by using a failover network that is activated during migration.
+|Q35 Chipset with UEFI |Virtual machines can use the Q35 chipset and UEFI, with or without SecureBoot.
```

Comment:
Did you mean to add Bochs and UEFI? You removed them in https://github.com/oVirt/ovirt-site/pull/2896. Rebase your branch against master. That should take care of it.